dashboard/app/controllers/levels_controller.rb

```diff
@@ -167,12 +167,17 @@ def embed_blocks
     authorize! :read, :level
     level = Level.find(params[:level_id])
     block_type = params[:block_type]
+    blocks = level.properties[block_type]
+    if level.game.app == 'eval'
```

Comment:
A possible way of genericizing this a bit—but not sure if it'd be worth it or if we'd have another use case coming down the line—

This if statement block becomes: `blocks = level.transform_for_embed(blocks)`

in `blockly.rb`:

```
def transform_for_embed(blocks)
  return blocks
end
```

in `eval.rb`:

```
def transform_for_embed(blocks)
  return # blocks regex
end
```